### ISSUE TITLE
Do not format graphql/multiline text

### ISSUE THREAD

**Issue by nboisnea** [karatelabs/intellij-plugin#69]
When testing GraphQL queries and more generally when writing multiline text like so:
```cucumber
Given text query =
"""
{
  myquery {
    field1
    field2
    field3
  }
}
"""
And request { query: '#(query)' }
When method post
Then status 200
```
Then formatting the code will change the text like this which is really not convenient:
```cucumber
"""
{
myquery {
field1
field2
field3
}
}
"""
```
Maybe reformatting should ignore whatever is inside triple quotes?

**Comment by ptrthomas** [comment]
> Maybe reformatting should ignore whatever is inside triple quotes?

@nboisnea agreed, we will work on this. thanks for the report !

**Comment by ptrthomas** [comment]
fixed in 2.0.7 available now
